scripts/build/cc/100-gcc.sh

```diff
@@ -403,10 +403,12 @@ do_gcc_core_backend() {
         extra_config+=("--with-system-zlib")
     fi
 
-    # Some versions of gcc have a deffective --enable-multilib.
+    # Some versions of gcc have a defective --enable-multilib.
```

Comment:
English is my 2nd language too, but:
deficient: not having enough of a specified quality or ingredient
defective: imperfect or faulty
Seems like defective is more appropriate (unless we want to state that some versions of GCC don't have enough multilibs) :)